kod/object/active/holder/room/monsroom/desertafterlife/makefile

```diff
@@ -0,0 +1,12 @@
+#
+# Makefile for compiling the Blakod
+#
+
+!include $(TOPDIR)\common.mak
+
+DEPEND = ..\desertafterlife.bof
+
+BOFS = desertaldunes.bof desertaldunes2.bof desertaldunes3.bof \
```

Comment:
There's a mismatch here between the listed bof "desertaldunes.bof" and the kod file "desertaldunes1.kod". This should have given you an error saying something like "AllocateObject can't find class id 12345" when you ran recreateall after building this locally. Even if the fixes/changes seem small, you should still be building and testing each one to catch things like this (ideally instead of me spending 20 minutes hunting down the bug on 104).